src/BarChart/components/BarChartContainer.ts

```diff
@@ -23,6 +23,7 @@ export default class BarChartContainer extends Component<BarChartContainerProps,
     };
     private subscriptionHandle?: number;
     private defaultColors: string[] = [ "#2CA1DD", "#76CA02", "#F99B1D", "#B765D1" ];
+    private intervalHandler?: number;
```

Comment:
Inaccurate name... It's an interval ID, not handler...

Reply:
Changed for all